pkg/cmd/cli/restore/delete.go

```diff
@@ -19,29 +19,147 @@ package restore
 import (
 	"fmt"
 
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	kubeerrs "k8s.io/apimachinery/pkg/util/errors"
+
+	arkv1api "github.com/heptio/ark/pkg/apis/ark/v1"
 	"github.com/heptio/ark/pkg/client"
 	"github.com/heptio/ark/pkg/cmd"
+	"github.com/heptio/ark/pkg/cmd/cli/common"
+	"github.com/heptio/ark/pkg/cmd/util/flag"
+	clientset "github.com/heptio/ark/pkg/generated/clientset/versioned"
 )
 
+// NewDeleteCommand creates and returns a new cobra command for deleting restores.
 func NewDeleteCommand(f client.Factory, use string) *cobra.Command {
+	o := &DeleteOptions{}
 	c := &cobra.Command{
-		Use:   fmt.Sprintf("%s NAME", use),
-		Short: "Delete a restore",
-		Args:  cobra.ExactArgs(1),
-		Run: func(c *cobra.Command, args []string) {
-			arkClient, err := f.Client()
-			cmd.CheckError(err)
+		Use:   fmt.Sprintf("%s [NAMES]", use),
+		Short: "Delete restores",
+		Example: `# delete a restore named "restore-1"
```

Comment:
this first line is misaligned with the others in the CLI help -- I'd add a tab between the ``` and the start of the text (see `pkg/cmd/cli/backup/delete.go` for an example)

Reply:
Fixed 👍